### PR TITLE
RemoveEmptyJavaDocParameters should remove "empty" lines #98

### PR DESCRIPTION
**Pull request by alixina9**
## What's changed?

I added fixes for issue #98  regarding removing trailing empty lines and other fixes . Also updated tests and added new ones with the expected cases of what I thought made more sense.

 ## What's your motivation? 

Issue #98 

 ## Anyone you would like to review specifically? 

@timtebeek maybe 🙂 

### Checklist
- [x] I've added unit tests to cover both positive and negative cases
- [x] I've added the license header to any new files through `./gradlew licenseFormat`
- [x] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

File: src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java [hunk 1]
```diff
@@ -222,12 +249,244 @@ void method() throws IllegalStateException {
...
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void issueTest() {
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java:377]: Perhaps good to reference the issue here, such that we have a link to the context of what prompted this test when reading the code.
```suggestion
    @Issue("https://github.com/openrewrite/rewrite-static-analysis/issues/98")
    void issueTest() {
```

**Comment by timtebeek** [comment]
Thanks a lot for getting this started @alixina9 ! I'd missed your PR earlier as I was traveling; from looking at your changes now I agree with when and where you'd like to see changes. There have been some changes to the main branch since though; would you be able to resolve the conflicts? You know best what you had intended. And feel free to start on the required code changes as well; and let us know if you need any help!

**Comment by alixina9** [comment]
> Thanks a lot for getting this started @alixina9 ! I'd missed your PR earlier as I was traveling; from looking at your changes now I agree with when and where you'd like to see changes. There have been some changes to the main branch since though; would you be able to resolve the conflicts? You know best what you had intended. And feel free to start on the required code changes as well; and let us know if you need any help!

I tried my best with resolving the conflicts, there were quite a few differences and I was already way on my way to fixing this issue 😄 If you have any suggestions to change things, let me know.

**Comment by timtebeek** [comment]
I want to thank you again for the effort you've put in here @alixina9 ; I'm sorry to have to admit I did not get to a timely review, and @koppor has since gone on to create #300 with a smaller changeset that was easier to review and see through. I hope this has not discouraged you from using or even contributing to the project in the future. Know that I feel bad for how long you've had to wait for feedback on this PR, only for it to be closed in the end for a different approach. It's happened to me in the past as well, and I know that can sting. Wishing you all the best there, and hope to see you around.

**Comment by koppor** [comment]
I think, @alixina9 also added "other fixes" regarding missing space between `*` and `@param`. Think, that could be discussed. However, I think, that autoformat is already taking care of that?

**Pull request closed without merging**